### PR TITLE
small cleanup

### PR DESCRIPTION
**Pull request by MCWertGaming**
This PR some smaller things left over from #132.

### CODE REVIEW

File: cpp-terminal/base.cpp [hunk 1]
```diff
@@ -6,13 +6,13 @@
 
 
 std::string Term::color(style value) {
-    return "\033[" + std::to_string(static_cast<unsigned int>(value)) + 'm';
+    return "\033[" + std::to_string((unsigned int)value) + 'm';
```
Comment (certik) [inline cpp-terminal/base.cpp:9]: We should not use C casts in C++. Let's keep using the `static_cast`.

Comment (MCWertGaming) [reply cpp-terminal/base.cpp:9]: Ok sure!

File: cpp-terminal/base.cpp [hunk 2]
```diff
@@ -33,39 +33,39 @@ void Term::write(const std::string& s) {
     std::cout << s << std::flush;
 }
 
-std::string Term::cursor_off() {
+const char* Term::cursor_off() {
     return "\x1b[?25l";
 }
```
Comment (certik) [inline cpp-terminal/base.cpp:38]: We should not use `char*` in C++, unless we have to. Let's use `std::string`.

Comment (MCWertGaming) [reply cpp-terminal/base.cpp:38]: I just thought we can skip ther conversion. The rueturn is actually a const char pointer after all. But I'll revert that one as well.

File: examples/menu_window_24bit.cpp [hunk 3]
```diff
@@ -51,7 +51,6 @@ std::string render(Term::Window_24bit& scr,
 int main() {
     try {
         Terminal term(true, true, false);
-        Term::save_screen();
```
Comment (certik) [inline examples/menu_window_24bit.cpp:54]: Is the screen saved automatically?

Comment (MCWertGaming) [reply examples/menu_window_24bit.cpp:54]: Yes, I have moved it into the constructor and destructor of the Terminal class. Basically the first `true` of it's parameters turns it on. I think it's much cleaner that way.

Comment (certik) [reply examples/menu_window_24bit.cpp:54]: Perfect.

**Review by certik — APPROVED** [review]
I think this is fine.

Before you merge, can you please create a better commit history? You can either squash it to one commit, or have more commits as needed, and have a nice git commit log. 

For example, you can have one commit which says "use single quotes instead of double quotes", and another commit which says "Remove save_screen(), already handled by the constructor". You can write more, but at least such one line should be part of every commit. Commits like "small cleanup" are fine for development, but one should fix that before merging.

**Comment by MCWertGaming** [comment]
Yeah sure, I'll make it like that next time.

**Comment by certik** [comment]
@MCWertGaming I asked if you could please do it this time. Since you already merged, let's leave it like that. But please let's not merge anymore commits like "small cleanup".

**Comment by MCWertGaming** [comment]
> @MCWertGaming I asked if you could please do it this time. Since you already merged, let's leave it like that. But please let's not merge anymore commits like "small cleanup".

I have squashed the comits like you said. The problem with changing commits afterward is that I'm not sure how you do it without making a new commit history by yourself and pushing it without making a force push (which creates a big warning in github). @certik

**Comment by certik** [comment]
Ah I missed that. You are right. The squash commit is ok. Sorry about that.

Yes, when I do it manually, I rewrite the commits locally and force push it into the branch. I mostly use gitlab, where it doesn't generate a warning. What warning does it generate on github? The alternative is to open a new PR with the polished commits (but that seems like an overkill for a case like this).

**Pull request merged**